scapy/contrib/modbus.py

```diff
@@ -571,6 +535,162 @@ class ModbusPDU2B0EReadDeviceIdentificationError(Packet):
                    ByteEnumField("exceptCode", 1, _modbus_exceptions)]
 
 
+_reserved_funccode_request = {
+    0x08: '0x08 Unknown Reserved Request',
```

Comment:
I don't really like leaving some work to do like this... Can't you find the fields ?